doc/source/myrocks/index.rst

```diff
@@ -0,0 +1,34 @@
+.. _myrocks_intro:
+
+============================
+Percona MyRocks Introduction
+============================
+
+`MyRocks <http://myrocks.io>`_ is a storage engine
+for `MySQL <https://www.mysql.com>`_ based on `RocksDB <http://rocksdb.org/>`_,
+an embeddable, persistent key-value store.
+*Percona MyRocks* is an implementation
+for `Percona Server <https://www.percona.com/software/percona-server>`_.
+
+The RocksDB store is based on the log-structured merge-tree (or LSM tree).
+It is optimized for fast storage
+and combines outstanding space, write, and read efficiency.

```

Comment:
outstanding space and write efficiency with acceptable read performance

Reply:
👍